src/v2/Apps/Order/Components/__tests__/SavedAddresses.jest.tsx

```diff
@@ -109,12 +113,11 @@ describe("SavedAddresses in collector profile", () => {
         addressConnection: mockAddressConnection,
       }),
     })
-    const editAddressComponent = wrapper.find("[data-test='editAddress']").at(0)
-    expect(editAddressComponent).toHaveLength(1)
+    const button = wrapper.find(Button)
```

Comment:
Blocking: 

Can we target this button more precisely than `Button`? This test could break pretty easily by adding new components to the page.

Reply:
Yes, you are right. Done)